src/lwc/gsChecklistItem/gsChecklistItem.js

```diff
@@ -125,13 +125,11 @@ export default class GsChecklistItem extends NavigationMixin(LightningElement) {
         target.disabled = true;
 
         updateCheckItem({'itemId': this.item.id, 'isChecked': checked})
-        .then (_ => {
+        .then ( _ => {
```

Comment:
I had to read up on this syntax (` _ => {}`) as it's not one I've seen before. I'm assuming adding the space before the underscore was not part of the fix in this PR?

Reply:
If I'm not mistaken this would also work as `() => {}`, which is how I've generally written these.